### DISCUSSION TITLE
[.NET 6 mobile] How to reference a Resource Dictionary in another project

### DISCUSSION THREAD

**Discussion by jonginleee** [unoplatform/uno#9061] [Q&A] [answered]
```
<ResourceDictionary>
    <ResourceDictionary.MergedDictionaries>
        <XamlControlsResources xmlns="using:Microsoft.UI.Xaml.Controls" />
        <ResourceDictionary Source="ms-appx:///Blossom/Resources/BlossomResource.xaml"/>
    </ResourceDictionary.MergedDictionaries>
</ResourceDictionary>
```

I referenced the Resource Dictionary in another project as above. It works fine on Windows, but an error occurs on mobile such as iOS and Android.

**System.Reflection.TargetInvocationException:** 'Exception has been thrown by the target of an invocation.'

Is there any way to solve this issue?

**Comment by jeromelaban** [comment]
Thanks for the report. You're likely getting another exception inside the `TargetInvocationException`. Could you post what it is?

**Comment by jonginleee** [comment, marked as answer]
Thanks for the reply.

I could reproduce this issue in an empty project. There was a issue when uploading, so I recreated the project again and the issue disappeared. In the previous project, the Build action of the Resource Dictionary was set to Content. Changing the build action to page solved the issue.
